src/fileops.rs

```diff
@@ -49,6 +52,8 @@ impl Drop for Fd {
 #[test]
 // Check open/close on a file that should exist in most linux based OS.
 fn open_close_devnull() -> () {
-  let fd = Fd::open("/dev/null", libc::O_RDONLY);
-  assert!(fd.is_some());
+  match Fd::open("/dev/null", libc::O_RDONLY) {
+    Err(e) => panic!(e.description().to_owned()),
```

Comment:
I think I saw panic in a test example a while back, is there a better way to say the test failed?